hadoop-ozone/dist/pom.xml

```
@@ -105,6 +105,16 @@
                   <artifactId>protobuf-java</artifactId>
                   <version>${grpc.protobuf-compile.version}</version>
                 </artifactItem>
+                <artifactItem>
+                  <groupId>org.apache.hadoop</groupId>
+                  <artifactId>hadoop-auth</artifactId>
+                  <version>2.7.3</version>
```

Try to use `hadoop2.version` instead

```suggestion
                  <version>${hadoop2.version}</version>
```

---

@nishitpatira I can merge the change once you address @smengcl's comments

---

Made the changes. 
Running CI in https://github.com/nishitpatira/ozone/actions/runs/3753493660